boards/nucleo-wl55jc/Kconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config BOARD
+    default "nucleo-wl55jc" if BOARD_NUCLEO_WL55JC
+
+config BOARD_NUCLEO_WL55JC
+    bool
+    default y
+    select BOARD_COMMON_NUCLEO64
+    select CPU_MODEL_STM32WL55JC
+
+    # Put defined MCU peripherals here (in alphabetical order)
+    select HAS_PERIPH_ADC
+    select HAS_PERIPH_I2C
+    select HAS_PERIPH_PWM
+    select HAS_PERIPH_RTT
+    select HAS_PERIPH_SPI
+    select HAS_PERIPH_TIMER
+    select HAS_PERIPH_UART
+
+    # Put other features for this board (in alphabetical order)
+    select HAS_RIOTBOOT
+
+    # Clock configuration
+    select BOARD_HAS_HSE
+
```

Comment:
This is not needed since `BOARD_HAS_HSE` is already selected in the common nucleo64 Kconfig which is included just below:

https://github.com/RIOT-OS/RIOT/blob/a8e9fa6253bc8275111a292b110d108825e6ce71/boards/common/nucleo64/Kconfig#L12-L14